include/runtime/header.h

```diff
@@ -75,6 +75,11 @@ extern "C" {
     void *f;
     char *g;
   } list;
+
+  typedef struct floating {
+    uint64_t exp;
```

Comment:
What is the meaning of this field? Is it the mpfr "exponent range"? An undocumented field named "exp" sounds like the value of the exponent.